tasks/instagrampoll.php

```diff
@@ -60,134 +60,137 @@ public function cleanup()
 
 	public function subscriptions()
 	{
-		$account = \Propeller\Instagram\Model_Account::query()
+		try 
+		{
+			$account = \Propeller\Instagram\Model_Account::query()
 				->where('active', 1)
 				->get_one();
 
-		$instagram = new \Instagram\Instagram;
-		$instagram->setAccessToken($account->token);
-
-		$subscriptions = \Propeller\Instagram\Model_Subscription::query()
-			->where('status', 'Live')
-			->get();
+			$instagram = new \Instagram\Instagram;
+			$instagram->setAccessToken($account->token);
 
-		foreach($subscriptions as $sub) {
+			$subscriptions = \Propeller\Instagram\Model_Subscription::query()
+				->where('status', 'Live')
+				->get();
 
-			$tag = $instagram->getTag($sub->object_id);
-			$media = $tag->getMedia();
-			$count = 0;
+			foreach($subscriptions as $sub) 
+			{
+				$tag = $instagram->getTag($sub->object_id);
+				$media = $tag->getMedia();
+				$count = 0;
 			
-			foreach($media as $med) {
-
-				$image = \DB::select('instagram_id')
-					->from(\Propeller\Instagram\Model_Image::table())
-					->where('instagram_id', $med->id)
-					->execute()
-					->as_array('instagram_id', null);
-										
-				try
+				foreach($media as $med) 
 				{
-					$ch = curl_init();
-					curl_setopt($ch, CURLOPT_URL, $med->images->standard_resolution->url);
-					curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-					$output = curl_exec($ch);
-					$content_type = curl_getinfo($ch, CURLINFO_CONTENT_TYPE);
-					curl_close($ch);	
+				
+					$image = \DB::select('instagram_id')
+						->from(\Propeller\Instagram\Model_Image::table())
+						->where('instagram_id', $med->id)
+						->execute()
+						->as_array('instagram_id', null);
+										
+						$ch = curl_init();
+						curl_setopt($ch, CURLOPT_URL, $med->images->standard_resolution->url);
+						curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+						$output = curl_exec($ch);
+						$content_type = curl_getinfo($ch, CURLINFO_CONTENT_TYPE);
+						curl_close($ch);	
 					
-					if($content_type == 'application/xml' and $xml = new \SimpleXMLElement($output))
-					{
-						// Handle access denied error
-						if($xml->Code == 'AccessDenied')						
-						{						
-							throw new ImageAccessDeindedException("Access denied to 'http://distilleryimage0.s3.amazonaws.com/47040426c57211e2880f22000a1f9ca7_5.jpg'. This entry hasn't been saved to the DB.");
-						}
+						if($content_type == 'application/xml' and $xml = new \SimpleXMLElement($output))
+						{
```

Comment:
Could we keep all braces for anything other than functions and classes on the same line please?